src/Aspire.Hosting.SqlServer/SqlServerBuilderExtensions.cs

```diff
@@ -112,11 +112,6 @@ public static IResourceBuilder<SqlServerServerResource> WithDataBindMount(this I
         {
             var path = Path.Combine(source, dir);
 
-            if (!Directory.Exists(path))
```

Comment:
I'm not sure if this warrants a breaking change, but we should add docs to tell users what they need to do to make this work on Linux.